### PR TITLE
feat(TMRX-1580):  Add lazy loading to slice images

### PR DESCRIPTION
**Pull request by bibicollins1**
### Description

Add lazy loading prop to slice images that are not the lead images.

[TMRX-1580](https://nidigitalsolutions.jira.com/browse/TMRX-1580)

### Checklist

- [x] Have you done any manual testing?
- [ ] Does it have automated tests?
- [ ] Do you need any other PRs merged before this (if so please list)?
- [ ] Do you need to update the README/Runbook
- [ ] Have you checked for [accessibility](https://nidigitalsolutions.jira.com/wiki/spaces/TNLDigital/pages/3898048523/Accessibility+Dev+Guide#Checklist)?

[TMRX-1580]: https://nidigitalsolutions.jira.com/browse/TMRX-1580?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by milanharia-tm — APPROVED** [review]

File: packages/ts-newskit/src/components/slices/article/index.tsx [hunk 1]
```diff
@@ -149,8 +149,13 @@ export const Article = ({
           )}
         </GridLayoutItem>
       )}
-
-      {hasImage && !hideImage && <CardMediaComponent {...cardImage} />}
+      {showImage ? (
+        isLeadImage ? (
+          <FullWidthCardMediaMob {...cardImage} />
+        ) : (
+          <CardMedia {...{ ...cardImage, loading: 'lazy' }} />
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/slices/article/index.tsx:156]: think `loading` is on the `media` prop;
e.g. `<CardMediaComponent {...cardImage} media={{loading: 'lazy'}} />`

Comment (bibicollins1) [reply packages/ts-newskit/src/components/slices/article/index.tsx:156]: @douglasmik tried that like: `<CardMedia media={{loading: 'lazy'}} { ...cardImage} />` but it's removed loading="lazy" from the snapshots.

Comment (bibicollins1) [reply packages/ts-newskit/src/components/slices/article/index.tsx:156]: This is adding loading onto the media prop - cardImage is the media prop passed through

**Review by douglasmik — APPROVED** [review]

File: packages/ts-newskit/src/components/slices/article/index.tsx (hunk 1 above)
Comment (adamosborne-tnl) [reply packages/ts-newskit/src/components/slices/article/index.tsx:156]: sorry, did you mean this is passing the prop down to the  tag, i don't see it?

Comment (bibicollins1) [reply packages/ts-newskit/src/components/slices/article/index.tsx:156]: So the CardMedia component get's it's prop data from cardImage which is defined above. This adds loading: lazy to that.

**Pull request merged**